README.md

```diff
@@ -57,36 +86,26 @@ Cylc is used to encode the entire pipeline from start to finish and relies on th
    - [ ] `conda env create -f ./config/ift-env.yaml`
    - [ ] `conda activate ift-env`
 
-3. Register an account with [space-track.org](https://www.space-track.org/) for SOIT
-
-4. Export SOIT username/password to environment variable
-   - [ ] From your home directory `nano .bash_profile`
-   - [ ] add `export HISTCONTROL=ignoreboth` to the bottom of your .bash_profile
-        * this will ensure that your username/password are not stored in history
-        * when exporting the following environment variables, there __must__ be a space in front of each command
-   - [ ] ` export SPACEUSER=<firstname>_<lastname>@brown.edu`
-   - [ ] ` export SPACEPSWD=<password>`
+3. Make sure you have registered for an account with `space-track.org` and exported your SOIT credentials as an environment variable on Oscar as outlined in in the [SOIT integration](#soit-integration) section.
 
-5. Prepare the runtime environment 
+4. Prepare the runtime environment 
 
     Cylc will use software dependencies inside a Singularity container to fetch images and satellite times from external APIs. 
    - [ ] It is a good idea to reset the Singularity cache dir as specified [here](https://docs.ccv.brown.edu/oscar/singularity-containers/building-images)
 
-   - [ ] first update the parameters at the top of the `flow.cylc` file:
-     - param_set # just enter `0` for one set of parameters or `0..n` for cycling
-     - startdate
-     - enddate
-     - crs
-     - bounding_box
-     - centroid_x #lat wgs84
-     - centroid_y #lon wgs84
-     - minfloearea
-     - maxfloearea
-     - project_dir
-
-**Note:** bounding box format = top_left_x top_left_y bottom_right_x bottom_right_y (x = lat(wgs84) or easting(epsg3413),  y = lon(wgs84) or northing(epsg3413))
+   - [ ] first populate the `flow.cylc` file by running: 
+   ```python
+   python workflow/scripts/flow_generator.py \
+   --csvfile "./config/site_locations.csv" \
+   --template "flow_template_hpc.j2" \
+   --template_dir "./config/cylc_hpc" \
+   --crs "<crs>" \
+   --minfloearea <value> \
+   --maxfloearea <value>
+   ```
+Run `python workflow/scripts/flow_generator.py --help` for a list of options.
 
-**Note:** if cycling through more than one set of parameters, enter values separated by a comma, e.g., `startdate = 2022-05-04,2022-05-08`
+**Note:** if cycling through more than one set of parameters, values need to be separated by a comma, e.g., `startdate = 2022-05-04,2022-05-08`
```

Comment:
I was on the fence as to whether we should delete or leave this, in case there's a use case where someone wants to manually fill out the flow file. But, you are right, this is in the Oscar section and might be misleading.